configure.ac

```diff
@@ -847,9 +847,12 @@ AC_ARG_WITH([crypto],
 AC_ARG_WITH([frr-format],
   AS_HELP_STRING([--with-frr-format[=<.../frr-format.so>]], [use frr-format GCC plugin]))
 
-AC_ARG_ENABLE([version-build-config],
+AC_ARG_ENABLE([version_build_config],
```

Comment:
This seems like unrelated, a separate commit needed?

Reply:
It is a pretty minor fix with a rarely used option that doesn't affect functionality at all, I figured I will just sneak it in here.

Reply:
yeah I really hate having fixes snuck in with other stuff.  If we have to back out one or more this makes it more difficult.  I agree with Donatas, put this in a separate commit.

Reply:
dropped it.

Reply:
moved to #16633